src/internal/repositories/horarioRepositorio/horarioRepositorio.go

```diff
@@ -0,0 +1,82 @@
+package horarioRepositorio
+
+import (
+	"database/sql"
+	"log"
+
+	"github.com/D-D-EINA-Calendar/CalendarServer/src/internal/core/domain"
+	_ "github.com/go-sql-driver/mysql"
+)
+
+type repo struct {
+	db *sql.DB
+}
+
+func New() *repo {
+	db, _ := sql.Open("mysql", "user:user@tcp(127.0.0.1:6033)/app_db")
+	return &repo{db}
+}
+
+type AuxAvaiableHours struct {
```

Comment:
Tienes que crear un paquete models igual que yo Domain donde poner todos tus tipos y una clase que dado 1 a N strcuts de tipo domain se transfromen en lo que quieres. Por ejemplo, `model.Room.from(domain.Room)` donde model.Room es es strucy que guardaras en la bbdd.

Reply:
Igual que la de from haz la de  "to".